linux/ibus-keyman/src/test/meson.build

```diff
@@ -3,7 +3,7 @@ keymanutil_sources = [
   util_files,
 ]
 
-keymanutil_deps = [ibus, gtk, x11, json_glib, kmnkbp_lib]
+keymanutil_deps = [ibus, gtk, json_glib, kmnkbp_lib]
```

Comment:
Does this need systemd?

Reply:
No. We don't have any tests in this directory that access dbus (which is the reason for the systemd dependency).